src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -137,7 +149,26 @@ g.test('it_is_invalid_to_draw_in_a_render_pass_with_missing_bind_groups')
   .fn(async t => {
     const { setBindGroup1, setBindGroup2, _success } = t.params;
 
-    const pipeline = t.createRenderPipeline();
+    const bindGroupLayouts = [
+      // bind group layout 0
+      [
+        {
+          binding: 0,
+          visibility: GPUShaderStage.VERTEX,
+          buffer: { type: 'uniform' },
+        },
+      ],
+
+      // bind group layout 1
+      [
+        {
+          binding: 0,
+          visibility: GPUShaderStage.VERTEX,
+          buffer: { type: 'uniform' },
+        },
+      ],
+    ] as Array<Array<GPUBindGroupLayoutEntry>>;
```

Comment:
note (no need to change this): I think `as const` would have also worked here, and it is more type-safe. `const bindGroupLayouts: GPUBindGroupLayoutEntry[][] =` might have also worked

Reply:
double checked this, `as const` cannot work due to our param declaration. So adopt the second form to ensure the type safe.